flashlight/autograd/Functions.cpp

```diff
@@ -272,6 +284,18 @@ Variable max(const double& lhsVal, const Variable& rhs) {
   return max(rhs, lhsVal);
 }
 
+Variable min(const Variable& input, const int& dim) {
+  auto result = min(input.array(), dim);
+  auto gradFunc = [result, dim](std::vector<Variable>& inputs,
```

Comment:
Hum, some code duplication here : the only diff seems to be the result array, right ?